embrace-android-core/src/test/java/io/embrace/android/embracesdk/internal/spans/EmbraceTracerTest.kt

```diff
@@ -347,19 +345,13 @@ internal class EmbraceTracerTest {
 
     private fun verifyPublicSpan(
         name: String,
-        traceRoot: Boolean = true,
         errorCode: ErrorCode? = null,
     ): EmbraceSpanData {
         val currentSpans = spanSink.completedSpans()
         assertEquals(1, currentSpans.size)
         val currentSpan = currentSpans[0]
         assertEquals(name, currentSpan.name)
         currentSpan.assertIsTypePerformance()
-        if (traceRoot) {
```

Comment:
We should probably keep this, but verify this via checking the parent of the span

Reply:
Updated assertion to check parentSpanId